src/main/java/fr/foodlimit/api/security/MockSecurityConfig.java

```diff
@@ -0,0 +1,48 @@
+package fr.foodlimit.api.security;
+
+import fr.foodlimit.api.security.jwt.JWTConfigurer;
+import fr.foodlimit.api.security.jwt.TokenProvider;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.HttpMethod;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.springframework.http.HttpMethod'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)